buddyboss/buddyboss-platform
# Only hidden groups in the site other group members single activity not accessible by other members

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1854

### How to test the changes in this Pull Request:

1. Make sure the site has all hidden groups.
2. Go to group A using member X and create an activity
3. Go to group A using member Y and view the single activity of X
4. See an error

### Proof Screenshots or Video
http://prntscr.com/vu2nbs



### Other information:
